frontend/package.json

```
@@ -144,7 +144,7 @@
     "js-base64": "^2.5.1",
     "js-yaml": "^3.13.1",
     "json-schema": "0.2.3",
-    "lodash-es": "4.17.x",
+    "lodash-es": "4.17.21",
```

We should use 4.7.21 or newer

```suggestion
    "lodash-es": "^4.17.21",
```